source/frontend/src/components/RegionManagement/DiscoverableRegions/DiscoverableAccountsTable.js

```diff
@@ -131,22 +132,24 @@ const DiscoverableAccountsTable = ({ selectedAccounts, onSelect }) => {
         header={
           <Header
             actions={
-              <SpaceBetween direction='horizontal' size='xs'>
-                <Button
-                  disabled={R.isEmpty(selectedAccounts)}
-                  onClick={() => setShowDeleteConfirm(true)}>
-                  Remove
-                </Button>
-                <Button
-                  loadingText='Removing'
-                  variant='primary'
-                  onClick={(e) => {
-                    e.preventDefault();
-                    history.push(IMPORT);
-                  }}>
-                  Import
-                </Button>
-              </SpaceBetween>
+              isUsingOrganizations()
+                  ? null
```

Comment:
nit: I think this should be undefined rather than null

Reply:
*done